src/cr/cube/cubepart.py

```diff
@@ -526,6 +539,22 @@ def rows_dimension_type(self):
     def rows_margin(self):
         return np.array([row.margin for row in self._matrix.rows])
 
+    @lazyproperty
+    def rows_proportions_moe(self):
+        """2D np.float64 ndarray of margin-of-error (MoE) for rows proportions.
+
+        The values are represented as percentage-fractions, analogue to the
+        `row_proportions` property. This means that the value of 3.5% will have the
+        value 0.035. The values can be np.nan when the corresponding percentage is also
+        np.nan, which happens when the respective table margin is 0.
+        """
+        return Z_975 * self.rows_std_err
+
+    @lazyproperty
+    def rows_std_err(self):
+        """2D np.float64 ndarray of standard errors for row percentages """
+        return np.sqrt(self._rows_variance / self.rows_margin[:, None])
```

Comment:
Question, have we planned to expose the `rows_std_err` ? In the previous PRD we only had the col_std_dev and col_std_err. If yes maybe we need the `rows_std_dev` too.